xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/filters/FilterList.vue

```diff
@@ -95,16 +95,9 @@ export default {
     // but if no Array is found, the string is parsed as a singleton
     // so that it always return an array
     values () {
-      try {
-        // Try to parse the string as an array
-        const values = JSON.parse(this.filterEntry.value || "[]");
-        if (values instanceof Array) {
-          return values;
-        } else {
-          return [values];
-        }
-      } catch (err) {
-        console.warn(err);
+      if (this.filterEntry.value) {
+        return this.filterEntry.value.split(",");
```

Comment:
Ah I didn't know it was actually possible to have a value with a comma!

When you say using an array as the filter value, do you mean:
- like the previous version (a stringified array as filter value):
```js
{ operator: "equals", value: "[\"option1\"]" }
```
- a true array object:
```js
{ operator: "equals", value: ["option1"] }
```

For now, I was considering that every filter values were string, because it is consistent (and easier to handle for storing the config in the url).
But if for some reason you prefer to use an array instead of a stringified array, I can change it.